src/optimizer.cls.php

```diff
@@ -136,6 +136,16 @@ public function serve($request_url, $file_type, $minify, $src_list)
 			File::save($tmp_static_file, $content, true, true);
 		}
 
+		// if CSS - run the minification on the saved file.
+		// Will move imports to the top of file and remove extra spaces.
+		if ($file_type == 'css') { 
+			$obj = new Lib\CSS_JS_MIN\Minify\CSS();
+			$obj->add(File::read($tmp_static_file));
+			$css = $obj->minify();
```

Comment:
Here you always minify the file regardless user settings? @timotei-litespeed

Reply:
Fixed in last push